core/blockchain.go

```diff
@@ -1588,20 +1589,9 @@ func (bc *BlockChain) writeBlockWithState(block *types.Block, receipts []*types.
 	// If the total difficulty is higher than our known, add it to the canonical chain
 	// Second clause in the if statement reduces the vulnerability to selfish mining.
 	// Please refer to http://www.cs.cornell.edu/~ie53/publications/btcProcFC.pdf
```

Comment:
This comment should be lifted over to ethash

Reply:
fixed